### PR TITLE
Add an initial CHANGES.md

### LINKED ISSUE #15: Establish ChangeLog file
This will be useful for the release announcement, and will help the users to see what has been changed.

### PR DESCRIPTION
**Pull request by fingolfin**
Resolves #15

### CODE REVIEW

**Comment by fingolfin** [comment]
Clearly the content could be better, but this is at least a start.

**Comment by DominikBernhardt** [comment]
Thanks @fingolfin . My question is how much we go into detail in the changeling. Do we mention certain things from the rewrite specifically or do you just want to leave it at that?

**Comment by fingolfin** [comment]
@DominikBernhardt I am really fine either way. I just wanted to get this started somehow. It's easy to add more details later on, if somebody wants to. Put another way: I am not interested myself in spending a lot of time writing a detailed account of what wen on. I won't object if somebody else wants to do that, though, and I'd be happy to review it, too :-).

That said, I only copied the "release notes" that somebody (you?) put on the website. Looking at it again, I'd actually prefer to not write "Major rewrite of the code...", so I just went in and rewrote the text. I plan no further changes to it at this time (other than to fix typos etc. you might find, of course ;-)

**Comment by gaehler** [comment]
We might mention some major milestones, which can be extracted from the history of the web pages. There was a version 2.1b1, which was used for a very long time, and which included some code cleanup, with adaptions to the requirements of the then current compilers and GMP library. But I won't get this done before Christmas. This can be added any time later.

**Comment by DominikBernhardt** [comment]
@gaehler I agree, these things can be added later. Also, do you think it's of much benefit so spent the time in going through the old pages? I mean, why would people like to read history release notes for old Carat versions?

File: CHANGES.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+This file describes changes in various CARAT versions.
+
+* X.Y (YYY-MM-DD)
+
+  - CARAT has been moved to GitHub.
+  - Dozens of bugs were fixed, including crashes, wrong results, data corruption
```
[the remark this answers is not in the document]
Comment (fingolfin) [reply CHANGES.md]: That's OK except it's not valid English; but "Many bugs..." would be fine by me :-)

[the remark this answers is not in the document]
Comment (DominikBernhardt) [reply CHANGES.md]: :man_facepalming: Sorry. I reckon we should go with "Many bugs" ;)

**Comment by fingolfin** [comment]
@DominikBernhardt updated

**Pull request merged**